assets/js/base/utils/price.js

```diff
@@ -5,25 +5,21 @@ import { sprintf } from '@wordpress/i18n';
 import { CURRENCY } from '@woocommerce/settings';
 
 /**
- * Format a price with currency data.
+ * Format a price, provided using the smallest unit of the currency, as a
+ * decimal complete with currency symbols using current store settings.
  *
- * @param {number} value Number to format.
- * @param {string} priceFormat  Price format string.
- * @param {string} currencySymbol Curency symbol.
+ * @param {number} price Price, in cents, to format.
  */
-export const formatPrice = (
-	value,
-	priceFormat = CURRENCY.price_format,
-	currencySymbol = CURRENCY.symbol
-) => {
-	if ( value === '' || undefined === value ) {
+export const formatPrice = ( price ) => {
+	if ( price === '' || price === undefined ) {
 		return '';
 	}
-	const formattedNumber = parseInt( value, 10 );
+
+	// eslint-disable-next-line @wordpress/valid-sprintf
 	const formattedValue = sprintf(
-		priceFormat,
-		currencySymbol,
-		formattedNumber
+		CURRENCY.priceFormat,
+		CURRENCY.symbol,
+		parseInt( price, 10 ) / 10 ** CURRENCY.precision
```

Comment:
Just a note because I wasn't aware until today, pow( 0 ) = 1, so we shouldn't see division by 0 errors.